### ISSUE TITLE
Fall back to installing the binary if OS is unsupported

### ISSUE THREAD

**Issue by aairey** [riemers/ansible-gitlab-runner#79]
Resorted to creating my own role since Fedora 30 and 31 do not have an RPM package available.
However this role only checks the distribution but not if the version is actually supported by Gitlab.

List of Operating Systems supported: https://docs.gitlab.com/runner/install/linux-repository.html

**Comment by aairey** [comment]
When trying this on an unsupported OS, it fails to download the RPM package from packagecloud.

**Comment by riemers** [comment]
Trying an unsupported OS and it fails is pretty much what you would want in the firts place no? How would you see something different in this case? If you have created a seperate role is this something we can integrate this? I actually have a custom one for inside the company i work since it doesn't have a connection to the internet.

**Comment by aairey** [comment]
Well, let me rephrase "Unsupported".
There is no package maintained for the OS, but any GNU/Linux is _supported_ by using the binary distribution.
As per: https://docs.gitlab.com/runner/install/

**Comment by riemers** [comment]
Well if you could incorporate your role in here too, we can always merge that.
